extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/URIExtractionNamespace.java

```diff
@@ -530,8 +531,10 @@ public JSONFlatDataParser(
       Preconditions.checkArgument(!Strings.isNullOrEmpty(valueFieldName), "[valueFieldName] cannot be empty");
       this.keyFieldName = keyFieldName;
       this.valueFieldName = valueFieldName;
+
+      // Copy jsonMapper; don't want to share canonicalization tables, etc., with the global ObjectMapper.
       this.parser = new DelegateParser(
-          new JSONParser(jsonMapper, ImmutableList.of(keyFieldName, valueFieldName)),
+          new JSONParser(jsonMapper.copy(), ImmutableList.of(keyFieldName, valueFieldName)),
```

Comment:
Thanks for the link. This should be fine then, it doesn't matter or not if we have injectable values for these parsers that just want to read `Map<String, String>`.